android-sdk-restclient/src/main/java/healthvault/client/MicrosoftHealthVaultRestApi.java

```diff
@@ -0,0 +1,1171 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.0.1.0
+ * Changes may cause incorrect behavior and will be lost if the code is
```

Comment:
The folder has to be named "healthvault.client" instead of generated, because this is the name space in the generated files

---
In reply to: [114874968](https://github.com/Microsoft/healthvault-java-sdk/pull/4#discussion_r114874968) [](ancestors = 114874968)